src/components/common/Search/Search.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable class-methods-use-this */
+import React, { Component } from "react";
+import propTypes from "prop-types";
+import { connect } from "react-redux";
+import { Form } from "semantic-ui-react";
+// import { browserHistory } from "react-router";
+import { searchMethod } from "../../../redux/actions/searchAction";
+import searchIm from "../../../assets/images/search.svg";
+
+export class SearchItems extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: "",
+      searchOption: ""
+    };
+    this.onChange = this.onChange.bind(this);
```

Comment:
Why????, we have es6 now. No need for ugly binding.

Reply:
Thank you I resolved this.